_includes/header.html

```diff
@@ -1,6 +1,7 @@
 <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
 <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 <link rel="stylesheet" type="text/css" href="css/bootstrap/bootstrap.css" />
+<link rel="stylesheet" type="text/css" href="css/bootstrap/bootstrap-theme.css" />
```

Comment:
@fmichonneau please take a look at my previews (https://github.com/swcarpentry/lesson-template/pull/182#issue-58113941) and decide whichever looks nicer: with Bootstrap theme or without.